src/store/tests/test-data/cat-adi-part3/test-data.cat-adi-part3.selector.ts

```diff
@@ -0,0 +1,3 @@
+import { Review, TestData } from '@dvsa/mes-test-schema/categories/ADI3';
+
+export const getReview = (state: TestData): Review => state.review;
```

Comment:
This selector is not necessary, if this is being used it should be referencing the reducer selector here ```src/store/tests/test-data/cat-adi-part3/review/review.reducer.ts```